tests/python_api/audio_unit_test.py

```diff
@@ -48,10 +55,23 @@ def plot_audio_wav(audio_tensor, idx):
     plt.savefig("output_folder/audio_reader/" + str(idx) + ".png")
     plt.close()
 
+def verify_non_silent_region_output(output_list, rocal_data_path, test_results, case_name):
+    ref_path = f'{rocal_data_path}/rocal_data/GoldenOutputsTensor/reference_outputs_audio/{case_name}_output.bin'
+    data_array = np.fromfile(ref_path, dtype=np.int32)
+    begin_nsr = output_list[0].detach().numpy()
+    length_nsr = output_list[1].detach().numpy()
+
+    if begin_nsr[0] == data_array[0] and length_nsr == data_array[1]:
+        print("PASSED!")
+        test_results[case_name] = "PASSED"
+    else:
+        print("FAILED!")
+        test_results[case_name] = "FAILED"
+
 def verify_output(audio_tensor, rocal_data_path, roi_tensor, test_results, case_name, dimensions):
     ref_path = f'{rocal_data_path}/rocal_data/GoldenOutputsTensor/reference_outputs_audio/{case_name}_output.bin'
```

Comment:
same as above. Why do we need 2 functions

Reply:
The verify NSR function is added because it needs two arguments not present in the verify_output function